signer/src/storage/model.rs

```diff
@@ -1,18 +1,24 @@
 //! Database models for the signer.
 
+use fake::faker::time::en::DateTimeAfter;
+
 /// Bitcoin block.
 #[derive(Debug, Clone, PartialEq)]
 #[cfg_attr(feature = "testing", derive(fake::Dummy))]
 pub struct BitcoinBlock {
     /// Block hash.
+    #[dummy(expr = "fake::vec![u8; 32]")]
     pub block_hash: BitcoinBlockHash,
     /// Block height.
     pub block_height: i64,
     /// Hash of the parent block.
+    #[dummy(expr = "fake::vec![u8; 32]")]
     pub parent_hash: BitcoinBlockHash,
     /// Stacks block confirmed by this block.
+    #[dummy(default)]
     pub confirms: Option<StacksBlockHash>,
     /// The time this block entry was created by the signer.
+    #[dummy(faker = "DateTimeAfter(time::OffsetDateTime::UNIX_EPOCH)")]
```

Comment:
Oh nice!

Reply:
Yeah, I stumbled upon it in the docs and figured it's nice to fix it while I'm at it :blush: